packages/foreman/rubygem-little-plugger/rubygem-little-plugger.spec

```
@@ -1,83 +1,91 @@
+# Generated from little-plugger-1.1.4.gem by gem2rpm -*- rpm-spec -*-
+# template: scl
 %{?scl:%scl_package rubygem-%{gem_name}}
 %{!?scl:%global pkg_name %{name}}
 
 %global gem_name little-plugger
 
-Summary: LittlePlugger is a module that provides Gem based plugin management
 Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 1.1.3
-Release: 23%{?dist}
+Version: 1.1.4
+Release: 1%{?dist}
+Summary: LittlePlugger is a module that provides Gem based plugin management
 Group: Development/Languages
 License: MIT
-URL: https://rubygems.org/gems/little-plugger
-Source0: http://gems.rubyforge.org/gems/%{gem_name}-%{version}.gem
-Requires: %{?scl_prefix_ruby}ruby(rubygems)
+URL: http://gemcutter.org/gems/little-plugger
```

This redirects now to the URL it used to have.
```suggestion
URL: https://rubygems.org/gems/little-plugger
```